apps/base-docs/docs/tools/onboarding.md

```diff
@@ -40,6 +41,14 @@ hide_table_of_contents: true
 
 ---
 
+## Openfort
+
+[Openfort](https://openfort.xyz) is an infrastructure provider designed to simplify the development of games and gamified experiences across their suite of API endpoints. Authenticated users can instantly access the embedded, non-custodial [smart account](https://www.openfort.xyz/docs/guides/accounts/smart) from within the game context and sign blockchain transactions.
```

Comment:
@joalavedra 

Could you also please add text here that clarifies to Base users how Openfort specifically supports Base as a network?